addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('mail.notification_alert', function (require) {
+"use strict";
+
+var widgetRegistry = require('web.widget_registry');
+var Widget = require('web.Widget');
+
+var notificationAlert = Widget.extend({
```

Comment:
`notificationAlert` => `NotificationAlert`
Also provide a short comment that describes what's the purpose of this widget.